test/unit/app/jobs/job_conf.sample_advanced.yml

```diff
@@ -749,6 +749,17 @@ execution:
       fallback_destination: local
       # Pick next available server and resubmit if an unknown error occurs
       # resubmit: {condition: 'unknown_error and attempt <= 3', environment: galaxycloudrunner}
+    vortex_dispatcher:
+      runner: dynamic
+      # Demonstrates how to use total-perspective-vortex, which enables dynamic destination
+      # selection for entities (Tools, Users, Groups) based on a configurable yaml file.
+      # See: https://total-perspective-vortex.readthedocs.io/ for documentation.
+      type: python
+      function: map_tool_to_destination
+      rules_module: vortex.rules
+      vortex_config_files:
+        - https://github.com/galaxyproject/total-perspective-vortex/blob/bf8ec46c672ed5a7ba2471d9b514cef15995bf3f/tests/fixtures/mapping-rules.yml
+        - config/vortex_rules_local.yml
```

Comment:
The first item should be removed (or commented out) and a corresponding file for the second line should be added that can be readily used with a local runner.

With the following `job_conf.yml`:
```
runners:
  local:
    load: galaxy.jobs.runners.local:LocalJobRunner
    workers: 4

execution:
  default: vortex_dispatcher
  environments:
    vortex_dispatcher:
      runner: dynamic
      type: python
      function: map_tool_to_destination
      rules_module: vortex.rules
      vortex_config_files:
        - config/mapping-rules.yml
    local:
      runner: local
```

A seemingly a minimal mapping file is as follows, jobs don't run (`'No destinations are available to fulfill request: default'`) so I'm guessing others will run into the same challenge of trying to understand how all fits together before seeing anything run:
```
global:
  default_inherits: default

tools:
  default:
    cores: 2
    mem: cores * 2

destinations:
  local:
    cores: 1
    mem: 2
    scheduling:
      prefer:
        - general

```

Reply:
Have added a dedicated sample file that should work for a local destination out of the box.